janusgraph-cql/src/test/resources/docker/update_config.py

```diff
@@ -16,9 +16,12 @@
 
 import os,re
 
-keystore = '/etc/ssl/test.keystore'
+keystore = '/etc/ssl/node.keystore'
 keystore_password = 'cassandra'
+truststore = '/etc/ssl/node.truststore'
+truststore_password = 'cassandra'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Possible hardcoded password: 'cassandra'](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=4611201)

Reply:
This hardcoded password issue already existed in `update_config.py` (line 20) - it seems like we want to configure an exception.  If that's correct, can someone point me in the right direction?

Reply:
You don’t have to set an exception. The tool isn’t a hard border for a pr to get merged.